hyper/client/aws/ec2.go

```diff
@@ -725,14 +780,27 @@ func StopServer(manifestPath string, remoteCfg HyperConfig.EC2RemoteConfiguratio
 		}
 		fmt.Println("Key Pair deleted:", keyName)
 
-		keyPath := path.Join(os.Getenv("HOME"), fmt.Sprintf("%s.pem", keyName))
-		_, err := os.Stat(keyPath)
-		if err == nil {
-			err = os.Remove(keyPath)
+		sshFolderPath := path.Join(UserHomeDir(), "/.ssh")
+
+		if keyName != ssh.DEFAULT_KEY {
+			originalDir, err := os.Getwd()
 			if err != nil {
-				panic("error deleting local pem file: " + err.Error())
+				panic("error changing working directory: " + err.Error())
+			}
+			os.Chdir(sshFolderPath)
+			_ = ssh.RemoveKeySshAgent(keyName)
+
+			_, err = os.Stat(keyName)
+			if err == nil {
+				err = os.Remove(keyName)
```

Comment:
Removed the delete key part and changed it for a message.
Also had to change the part during server start, it will search at .ssh for a private key named the same as the project and if it exists, this key will use it instead of creating a new one.